tests/web_platform_tests/test_request.py

```diff
@@ -93,3 +96,25 @@ def on_message():
     await test(404)
     await test(410)
     await test(503)
+
+
+@pytest.mark.asyncio
+async def test_request_post_to_connect():
+    """Test EventSource option method for connection.
+
+    """
+    source = sse_client.EventSource(
+        WPT_SERVER + 'resources/cache-control.event_stream?pipe=sub',
```

Comment:
this endpoint supports POST method
```suggestion
        WPT_SERVER + 'resources/message.py',
```